README.md

```diff
@@ -169,3 +169,17 @@ You could generate single app doc with this:
 ```sh
 $ gulp jsdoc:system
 ```
+
+## Preprocessor Parameters
+
+HTML, JavaScript and CSS can be preprocessed using `build/preprocessor.js`. See
+this file for example usage in a `build.js` file.
+
+Customise a specific distro using these flags with a `distro.mk` file in the
+`distros/<distro_name>` directory.
+
+ Flag           | Description
+----------------|----------------------------------------
+NO_LOCK_SCREEN  | Disable the Gaia lockscreen by default
+NO_BLUETOOTH    | Disable Bluetooth UIs
+FIREFOX_SYNC    | Enable Firefox Sync for FxOS
```

Comment:
please also add this section (with related parameter) in each affected app.
It will make per app (ex settings) debug easier without referencing cross over other modules.